web/templates/web/study-detail.html

```diff
@@ -83,7 +83,12 @@ <h4>{% trans "Compensation" %}</h4>
                     {% endif %}
                 </table>
                 <p class="pb-0">
-                    <em class="mt-3 mb-0 text-center text-muted">{% trans "This study is conducted by" %} {{ object.contact_info }}</em>
+                    <em class="mt-3 mb-0 text-center text-muted">
+                        {# Translators: Please include the untranslated "%(contact)s" string to mark where the contact information should be inserted. #}
+                        {% blocktranslate trimmed with contact=study.contact_info %}
+                            This study is conducted by {{ contact }}.
+                        {% endblocktranslate %}
```

Comment:
The note to the translators is nice, but don't they just see the PO files?

Reply:
Yes, they probably won't be looking at comments here. But this is a trick I learned from [this section](https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#comments-for-translators-in-templates) in the Django docs. If you have a comment that starts with the "Translators" keyword right before the `trans`/`blocktrans` template tag, it gets added to the corresponding msgid in .po file, like this:
> #. Translators: Please include the untranslated "%(contact)s" string to mark where the contact information should be inserted.
#: web/templates/web/study-detail.html:88
msgid "This study is conducted by %(contact)s."
msgstr "Diese Studie wird durchgeführt von %(contact)s."

(You can also leave [translator comments](https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#comments-for-translators) for text strings that are set in the python code rather than a template.)

For the most part I don't think we need to leave translator comments. But I think they are useful when we're incorporating arguments that need to be added with untranslated variable names and specific syntax, because this seems to be a common source of confusion for translators (moreso for EFP).